src/js/constants/AppFormErrorMessages.js

```diff
@@ -21,6 +21,10 @@ const applicationFieldValidationErrors = Util.deepFreeze({
     "Host Path must be a valid path",
     "Mode must not be empty"
   ],
+  containerVolumesLocal: [
+    "Size must be a non-negative Number",
+    "Contianer Path must be a valid path"
+  ],
   cpus: ["CPUs must be a non-negative number"],
   disk: ["Disk Space must be a non-negative number"],
   dockerImage: ["Image cannot  contain whitespaces"],
```

Comment:
Wenn Du schonmal dabei bist:
Could you get rid of the extra space after "connat". Thx.